PBM-498 Removed --mongodb-uri from samples, updated doc to set PBM-MO… (#382)
percona/percona-backup-mongodb: doc/source/running.rst

.. tip::
   
   Running as the ``mongod`` user would be the most intuitive and convenient way.
   But if you want it can be another user.

> The |pbm-agent| is connected to its mongod and started OK is
> confirmed by the *"pbm agent is listening for the commands"* message printed to the log
> file.

Reviewer: ```suggestion
When a message *"pbm agent is listening for the commands"* is printed to the
|pbm-agent| log file it confirms it connected to its mongod successfully.
```
